packages/muze/src/canvas/component-wrapper-maker.js

```diff
@@ -17,6 +31,21 @@ const headerMap = {
         configType: SUB_TITLE_CONFIG
     }
 };
+/**
+ * returns if message Component is needed or not
+ *
+ *
+ * @param {Object} renderDetails Extra details required for rendering the headers
+ * @param {Object} canvas canvas instance
+ * @return {Instance} Returns the respective wrappers
+ */
+const isGridPresent = (renderDetails) => {
+    const { rows, columns, values } = renderDetails.components;
+    if (!rows && !columns && !values) {
```

Comment:
Read about design pattern to change the way this code can be re-written. 

`if (!rows && !columns && !values) ` is not preferred